File: videosdk-live/videosdk-rtc-android-java-call-trigger-example/call-trigger-app/app/src/main/java/live/videosdk/call_trigger/example/Network/ApiService.java
package live.videosdk.call_trigger.example.Network;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;

public interface ApiService {

    @POST("/initiate-call")
    Call<String> initiateCall(@Body Map<String, Object> callRequestBody);

    @POST("/update-call")
    Call<String> updateCall(@Body Map<String,Object> callUpdateBody);
}

File: videosdk-live/videosdk-rtc-android-kotlin-call-trigger-example/call-trigger-app/app/src/main/java/live/videosdk/call_trigger/kotlin/example/Network/ApiService.kt
package live.videosdk.call_trigger.kotlin.example.Network

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface ApiService {

    @POST("/initiate-call")
    @JvmSuppressWildcards
    fun initiateCall(@Body callRequestBody: Map<String, Any>): Call<String>

    @POST("/update-call")
    @JvmSuppressWildcards
    fun updateCall(@Body callUpdateBody: Map<String, Any>): Call<String>
}
